### ISSUE TITLE
Add support for caqti 2.0

### ISSUE THREAD

**Issue by smorimoto** [roddyyaga/ppx_rapper#36]
As far as I understand correctly, I think we need to add support for caqti 2.0, right? @paurkedal

**Comment by smorimoto** [comment]
Apparently it was done in https://github.com/roddyyaga/ppx_rapper/pull/35.

**Comment by paurkedal** [comment]
Yes, there were backwards incompatible changes in Caqti 2.0.0 (2.0.1 first opam release).
